uui/components/tables/columnsConfigurationModal/ColumnsConfigurationModal.tsx

```diff
@@ -98,7 +82,7 @@ export function ColumnsConfigurationModal<TItem, TId, TFilter>(props: ColumnsCon
                 {renderGroupTitle(i18n.hiddenSectionTitle, amountHidden)}
                 <FlexRow cx={ styles.groupItems } size="30">
                     {groupedColumns.hidden.map((c) => (
-                        <ColumnRow column={ c } key={ c.key } />
+                        <ColumnRow column={ c } key={ c.key } renderItem={ props.renderItem ? () => props.renderItem(c) : null } />
```

Comment:
why do we have `props.renderItem ? () => props.renderItem(c) : null` condition here, if we have the same inside `ColumnRow`?

Reply:
fixed it